mono/mini/debugger-agent.c

```diff
@@ -9991,6 +9998,34 @@ string_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 	return ERR_NONE;
 }
 
+static ErrorCode
+pointer_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
+{
```

Comment:
command_set_to_string () etc needs to be modified to handle the new command set/commands.